apps/trading/components/market-selector/use-market-selector-list.spec.tsx

```diff
@@ -4,12 +4,12 @@ import {
   isMarketActive,
   useMarketSelectorList,
 } from './use-market-selector-list';
-import { Product } from './product-selector';
+import { Product } from '../../components/market-selector/product-selector';
 import { Sort } from './sort-dropdown';
 import { createMarketFragment } from '@vegaprotocol/mock';
 import { MarketState } from '@vegaprotocol/types';
 import { useMarketList } from '@vegaprotocol/markets';
-import type { Filter } from './market-selector';
+import type { Filter } from '../../components/market-selector/market-selector';
```

Comment:
with a module index file shorter path `'../../components/market-selector'` should also work fine